### PR TITLE
Add "Introducing Truffle Dashboard" post

### PR DESCRIPTION
**Pull request by kevinbluer**
Note both the page itself (and the reference on the top-level /blog page). A preview can be reached from the `fleek/build` link under the checks.

### CODE REVIEW

File: src/blog/introducing-truffle-dashboard/index.md [hunk 1]
```diff
@@ -8,158 +8,158 @@ hide:
 
 **By [Kingsley Arinze](https://twitter.com/heydamali)**
 
-One of our core missions at Truffle is to make the right developer tooling available to builders in the web3 space while putting security at the forefront. Among the many requirements smart contract developers have for their tools, they require that these tools are safe and are able to keep sensitive information secret, reducing their chances of being compromised during development. 
+One of our core missions at Truffle is to make the right developer tooling available to builders in the web3 space while putting security at the forefront. As smart contract developers, among the many requirements you have for your tools, you require tools to keep your sensitive information secret and safe. At Truffle, we always seek to reduce the chances of your security being compromised, regardless of where you are in the development lifecycle.
```
Comment (gnidan) [inline src/blog/introducing-truffle-dashboard/index.md:11]: I reworked this to be consistent with other second-person pronouns in the post. Maybe it's a bit awkward... thoughts?

File: src/blog/introducing-truffle-dashboard/index.md [hunk 2]
```diff
@@ -8,158 +8,158 @@ hide:
...
 
-The Truffle Dashboard was borne out of our years of experience working on developer tools for the Ethereum ecosystem and receiving valuable feedback from our users. We were excited to build this out. We hope you're as excited to try it out.
+The Truffle Dashboard was borne out of our years of experience working on developer tools for the Ethereum ecosystem and receiving valuable feedback from our users. We are extremely excited to release this feature. We hope you're excited to try it out.
 
-Download the Truffle Dashboard today by running the command `npm uninstall -g truffle && npm install -g truffle` as an existing Truffle user or `npm install -g truffle` as a new user. See documentation for more information on getting started.
+Download the latest version of Truffle and start using Truffle Dashboard today by running the command `npm uninstall -g truffle && npm install -g truffle` (as an existing Truffle user) or `npm install -g truffle` (as a new user).  Please refer to the [Truffle Dashboard](/docs/truffle/getting-started/using-the-truffle-dashboard.html) feature documentation for more.
```
Comment (gnidan) [inline src/blog/introducing-truffle-dashboard/index.md:161]: Maybe these two paragraphs should be swapped

File: src/blog/introducing-truffle-dashboard/index.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+With this setup, when you run the command `hardhat deploy --network truffle-dashboard`, the request is forwarded to the Dashboard where you can inspect and process them.
+
+## Conclusion ❤️️
+
+**With Truffle Dashboard, you no longer have to worry about giving your private keys to someone else's JavaScript code.** You can now relax and leave it up to MetaMask to keep your private keys safe and sign transactions on your behalf.
+
+The Truffle Dashboard was borne out of our years of experience working on developer tools for the Ethereum ecosystem and receiving valuable feedback from our users. We are extremely excited to release this feature. We hope you're excited to try it out.
+
+Download the latest version of Truffle and start using Truffle Dashboard today by running the command `npm uninstall -g truffle && npm install -g truffle` (as an existing Truffle user) or `npm install -g truffle` (as a new user).  Please refer to the [Truffle Dashboard](/docs/truffle/getting-started/using-the-truffle-dashboard.html) feature documentation for more.
```
Comment (gnidan) [inline src/blog/introducing-truffle-dashboard/index.md:161]: Should we switch these two paragraphs around?

File: src/blog/introducing-truffle-dashboard/index.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+    // ...
+
+    "truffle-dashboard": {
+      url: "http://localhost:24012/rpc"
+    }
+  },
+};
+```
+
+With this setup, when you run the command `hardhat deploy --network truffle-dashboard`, the request is forwarded to the Dashboard where you can inspect and process them.
```
Comment (gnidan) [inline src/blog/introducing-truffle-dashboard/index.md:153]:
```suggestion
With this setup, when you run the command `hardhat deploy --network truffle-dashboard`, the request is forwarded to Truffle Dashboard, where you can inspect and process them.
```

File: src/blog/introducing-truffle-dashboard/index.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+## Conclusion ❤️️
+
+**With Truffle Dashboard, you no longer have to worry about giving your private keys to someone else's JavaScript code.** You can now relax and leave it up to MetaMask to keep your private keys safe and sign transactions on your behalf.
+
+The Truffle Dashboard was borne out of our years of experience working on developer tools for the Ethereum ecosystem and receiving valuable feedback from our users. We are extremely excited to release this feature. We hope you're excited to try it out.
```
Comment (gnidan) [inline src/blog/introducing-truffle-dashboard/index.md:159]:
```suggestion
Truffle Dashboard was borne out of our years of experience working on developer tools for the Ethereum ecosystem and receiving valuable feedback from our users. We are extremely excited to release this feature. We hope you're excited to try it out.
```

File: src/blog/introducing-truffle-dashboard/index.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
+---
+title: Introducing Truffle Dashboard - Stop copy + pasting your private keys!
+hide:
+  - navigation
+---
+
+![truffle dashboard - banner](./truffle-dashboard-private-keys.png)
+
+**By [Kingsley Arinze](https://twitter.com/heydamali)**
+
```
Comment (gnidan) [inline src/blog/introducing-truffle-dashboard/index.md:10]: Do we need an "Introduction" H2 here? The ToC on the right looks a bit weird without it.

File: src/blog/introducing-truffle-dashboard/index.md (hunk 4 above)
Comment (kevinbluer) [reply src/blog/introducing-truffle-dashboard/index.md:153]: Looks good 👍

File: src/blog/introducing-truffle-dashboard/index.md (hunk 6 above)
Comment (kevinbluer) [reply src/blog/introducing-truffle-dashboard/index.md:10]: Agreed

File: src/blog/introducing-truffle-dashboard/index.md (hunk 1 above)
Comment (kevinbluer) [reply src/blog/introducing-truffle-dashboard/index.md:11]: I feel that the whole paragraph is a little verbose, a potential alternative: "_One of our core missions at Trufflesuite is to provide tooling that aids in the complete decentralized application (dapp) development lifecycle, while maintaining rigorous security practices throughout._"

**Review by gnidan — APPROVED** [review]
I only wrote ~30% of the words. Approved!

**Pull request merged**